app/lib/features/tasks/widgets/task_item.dart

```diff
@@ -11,47 +12,65 @@ import 'package:flutter_gen/gen_l10n/l10n.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:go_router/go_router.dart';
 import 'package:intl/intl.dart';
+import 'package:logging/logging.dart';
 import 'package:skeletonizer/skeletonizer.dart';
 
+final _log = Logger('a3::tasks::widget::task_item');
+
 class TaskItem extends ConsumerWidget {
-  final Task task;
+  final String taskListId;
+  final String taskId;
   final bool showBreadCrumb;
   final Function()? onDone;
   final Function()? onTap;
 
   const TaskItem({
     super.key,
-    required this.task,
+    required this.taskListId,
+    required this.taskId,
     this.showBreadCrumb = false,
     this.onDone,
     this.onTap,
   });
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return ListTile(
-      onTap: () {
-        context.pushNamed(
-          Routes.taskItemDetails.name,
-          pathParameters: {
-            'taskId': task.eventIdStr(),
-            'taskListId': task.taskListIdStr(),
+    return ref
+        .watch(taskItemProvider((taskListId: taskListId, taskId: taskId)))
+        .when(
+          data: (task) => ListTile(
+            onTap: () {
+              context.pushNamed(
+                Routes.taskItemDetails.name,
+                pathParameters: {
+                  'taskId': task.eventIdStr(),
+                  'taskListId': task.taskListIdStr(),
+                },
+              );
+            },
+            horizontalTitleGap: 0,
+            minVerticalPadding: 0,
+            contentPadding: const EdgeInsets.all(3),
+            visualDensity: const VisualDensity(horizontal: 0, vertical: -4),
+            minLeadingWidth: 35,
+            leading: leadingWidget(task),
+            title: takeItemTitle(context, task),
+            subtitle: takeItemSubTitle(ref, context, task),
+            trailing: trailing(ref, task),
+          ),
+          error: (error, stack) {
+            _log.severe('failed to load task', error, stack);
+            return const ListTile(
+              title: Text('failed'),
+            );
```

Comment:
we should show the error for easier debugging
```suggestion
            return ListTile(
              title: Text('Loading failed: $error'),
            );
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2046/commits/030800270c4512f1ac5892707630d8fdc0bc0b2d